ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.2.yaml

```diff
@@ -163,6 +163,7 @@ tests:
           cpu: 100m
           memory: 200Mi
     - ref: cucushift-chainupgrade-toimage
+    - ref: enable-qe-catalogsource
```

Comment:
Can we directly use `cucushift-installer-rehearse-aws-ipi` install workflow?

Reply:
We can't. `ipi-install-monitoringpvc` will fail install on 4.3 and earlier. Also `enable-qe-catalogsource` step is only available for the OCP4.5+.

Reply:
> ipi-install-monitoringpvc will fail install on 4.3 and earlier.

we can set `PERSISTENT_MONITORING: "false"` env var to skip the step.

>  Also enable-qe-catalogsource step is only available for the OCP4.5+.

Good to know, does `the OCP4.5+` include 4.5 or not?

Reply:
I tried it on 4.5 but [failed](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/48928/rehearse-48928-periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-4.15-upgrade-from-stable-4.5-aws-ipi-f30/1761955103373266944). So I extracted it in 4.5 to 4.15 job.

Reply:
From the [log](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/48928/rehearse-48928-periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-4.15-upgrade-from-stable-4.5-aws-ipi-f30/1761955103373266944/artifacts/aws-ipi-f30/gather-extra/artifacts/oc_cmds/clusteroperators), maybe we can try it again?

Reply:
Per slack thread with @jianzhangbjz, it supports 4.5 but the official support is starting from 4.6. Do you prefer to install it on 4.5?

Reply:
okay, if we already confirmed with that, then keep the current code.